src/main/kotlin/gg/skytils/skytilsmod/core/Config.kt

```diff
@@ -2886,6 +2900,7 @@ object Config : Vigilant(
     )
     var windHider = 0
 
+
```

Comment:
Where this spawn from

Reply:
idk how this got here
imma remove this lol